stable/ibm-istio/charts/tracing/templates/ingress-jaeger.yaml

```diff
@@ -1,4 +1,4 @@
-{{ if (.Values.jaeger.ingress.enabled) and eq .Values.provider "jaeger" }}
+{{ if (.Values.tracing.jaeger.ingress.enabled) and eq .Values.provider "jaeger" }}
```

Comment:
I don't think it's the right way to refer to the parameter, please check this example: https://github.com/IBM/charts/blob/master/stable/ibm-istio/charts/tracing/templates/deployment.yaml#L32